CppCoreGuidelines.md

```diff
@@ -15704,6 +15704,42 @@ Alternatively, we will decide that no change is needed and delete the entry.
 * Speaking of lambdas, what would weigh in on the decision between lambdas and (local?) classes in algorithm calls and other callback scenarios?
 * And speaking of `std::bind`, Stephen T. Lavavej criticizes it so much I'm starting to wonder if it is indeed going to fade away in future. Should lambdas be recommended instead?
 * What to do with leaks out of temporaries? : `p = (s1 + s2).c_str();`
+
+  * Can this example not be fixed in the library? E.g. `char const* c_str() const &; char const* c_str() &&=delete;`. gcc 5.3.1 on Fedora produces an error when given the code:
```

Comment:
This particular suggestion is meant to be in relation to the question "What to do with leaks out of temporaries?" rather than as a new comment.  Please suggest a way to mark such a relation.